scripts/user_env/0_nodenv.sh

```diff
@@ -12,15 +12,19 @@ git clone --depth 1 https://github.com/nodenv/nodenv-default-packages.git "${NOD
 
 ( cd ${NODENV_ROOT} && src/configure && make -C src )
 
-# install package managers in all nodenv versions
-echo "npm" >> "${NODENV_ROOT}/default-packages"
+# Install yarn package manager in all nodenv versions
+# Leave npm version aligned with version of node
 echo "yarn" >> "${NODENV_ROOT}/default-packages"
 
 # nodenv install
 NODE_VERSION=12.22.1
 eval "$(nodenv init -)"
 nodenv install "${NODE_VERSION}"
 nodenv global "${NODE_VERSION}"
-node --version
 
 nodenv package-hooks install --all
+
+# Show installed versions
+echo "node version = $(node --version)"
+echo "npm version = $(npm --version)"
+echo "yarn version = $(yarn --version)"
```

Comment:
Does this stuff matter? will only be seen in the docker build commands

Reply:
There was already a line for node so I just extended it. It wasn't easy in the log to find the node version so I just made them a but more explicit. We can always pull in the container to check this I agree, but I don't see it hurting.